.gitpod/Dockerfile

```diff
@@ -1,6 +1,13 @@
-FROM gitpod/workspace-full
-
+FROM gitpod/workspace-full:latest
+ARG JAVA_VERSION=21.0.6-tem
 ARG MAVEN_VERSION=3.9.9
-
-RUN brew install gh && \
-    bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh && sdk install maven ${MAVEN_VERSION} && sdk default maven ${MAVEN_VERSION}"
+# Install Java 21
+RUN bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh && \
```

Comment:
please collapse this back into one RUN layer
see: https://github.com/hadolint/hadolint/wiki/DL3059